pkgs/tools/X11/autokey/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "autokey";
+  version  = "0.94.0";
+  name = "${pname}-${version}";
```

Comment:
When using `buildPythonApplication` and `buildPythonPackage` the name attribute defaults to  `"${pname}-${version}"' so this live can be removed.